app/lib/features/search/widgets/quick_jump.dart

```diff
@@ -98,11 +109,9 @@ class QuickJump extends ConsumerWidget {
               },
               icon: const Icon(
                 Atlas.chats_thin,
-                size: 32,
```

Comment:
If I am not mistaken that was smaller on purpose, too. As seen in the image above, it is just too large for the circle it is in...